src/lib.rs

```diff
@@ -6,14 +6,15 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Interface to the random number generator of the operating system.
+//! Interface to the operating system's random number generator.
 //!
-//! # Platform sources
+//! # Supported targets
 //!
-//! | OS               | interface
+//! | Target           | Implementation
 //! |------------------|---------------------------------------------------------
```

Comment:
These aren't targets, but they are almost targets. Not sure what you should call it, but I think target isn't correct. You could change the column to be the taget triples. You could call it environment instead. 

This is a bit of a bikeshed and anyone reading this can probably read between the lines.

Reply:
We could make this table have three columns: Target, Target Triple, Implementation

would look like:

| Target | Target Triple | Implementation
| --- | ----- | --- | 
| Linux, Android | `*-linux-*` | [`getrandom`](1) system call if available, otherwise [`/dev/urandom`](2) after successfully polling `/dev/random` |
| Windows | `*-pc-windows-*` | [`RtlGenRandom`](3) |
| [Windows UWP](22) | `*-uwp-windows-*` | [`BCryptGenRandom`](23) | 

I'll need to see if this actually ends up looking good though